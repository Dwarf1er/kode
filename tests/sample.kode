SET X TO 1 PLUS 5.
TEE X.
TEE SET Y TO 2 PLUS X.
SET Z TO 2 PLUS X MINUS Y.
TEE X PLUS Y PLUS Z.
SET W TO TEE Z MINUS X.
SET W TO W PLUS 1.
TEE W.
SET X TO SET Y TO SET Z TO 0.