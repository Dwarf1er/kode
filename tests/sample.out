6
8
14
-6
-5
0